src/main/resources/category/java/remoting.xml

```diff
@@ -245,12 +245,14 @@ class Good {
     <rule name="HttpClientBuilderWithoutPoolSize"
           class="net.sourceforge.pmd.lang.rule.xpath.XPathRule"
           language="java"
-          message="HttpClient builder v4/v5 is used and the connection pool is not configured: defaults of 2/5 per route and 20/25 max total are probably not right (for v4/v5)."
+          message="HttpClient builder version 4, or version 5, is used and the connection pool is not configured: defaults per route (2 and 5 respectively) and max total (20 and 25 respectively) are probably not right."
```

Comment:
Too long, make it shorter without the specifics/details. Details in the description.